cmd/nimbess-cni.go

```diff
@@ -133,6 +158,12 @@ func cmdAdd(args *skel.CmdArgs) error {
 		return err
 	}
 
+	netns, err := ns.GetNS(args.Netns)
```

Comment:
why are you opening the namespace here, just to ensure it exists before you make the call to agent?

Reply:
Yes. Do you want to get rid of this?